app/components/satis/card/component.html.slim

```diff
@@ -1,67 +1,68 @@
 .sts-card.sts-tabs data-controller="satis-tabs" data-satis-tabs-persist-value="false" data-satis-tabs-key-value="#{key}"
-  - if header?
-    .sts-card__header class="#{tabs? ? '' : 'border-b border-gray-200'} #{header_background_color[:light]} dark:#{header_background_color[:dark]}"
-      .-ml-4.-mt-4.flex.justify-between.items-center.flex-wrap.sm:flex-nowrap
-        - if icon
-          .ml-4.mt-4.flex-shrink-0.text-primary-600.dark:text-gray-300
-            i class=icon
-        .ml-4.mt-4.flex-1
-          h3.text-lg.leading-6.font-medium.text-gray-900.dark:text-gray-300
-            == title
-          - if description.present?
-            p.mt-1.text-sm.text-gray-500.dark:text-gray-500
-              == description
+  div id="act_table_view" data-controller="act-table"
```

Comment:
Can we add a `html` attribute to the card which allows you to set additional attributes on the `.sts-card`  above, where we always merge `data-controller="satis-tabs"` in it.

We can't hardcode the act-table controller in satis like this.